Makefile

```diff
@@ -34,6 +34,8 @@ black_check:
 	black --check --diff $(PY_SOURCE) --exclude $(PY_SOURCE)/metadata/generated
 
 generate:
+	@echo "Installing Python Data Model Generator"
+	pip install datamodel-code-generator
```

Comment:
could we change this for `make install_dev`? We have the req in there with the fixed version. WDYT?

Reply:
Ohh, got it. I will remove the above code and ask users to do `make install_dev generate` in order to install the package dependencies as well.
Sounds good to me @pmbrull

Reply:
amazing thanks!!